core/src/main/java/com/sequenceiq/cloudbreak/controller/StackCreatorService.java

```diff
@@ -178,6 +182,7 @@ public StackV4Response createStack(User user, Workspace workspace, StackV4Reques
                 determineBlueprint(stackRequest, workspace),
                 LOGGER,
                 "Stack request converted to stack took {} ms for stack {}", stackName);
+            validateBlueprintVersion(blueprint, stackRequest);
```

Comment:
I'm not sure about the blueprint version, because QE sometimes use the same blueprint for multiple runtime versions.